datahub-web-react/src/app/homeV3/modules/assetCollection/AssetCollectionModal.tsx

```diff
@@ -59,10 +60,12 @@ const AssetCollectionModal = () => {
             submitButtonProps={{ disabled: isDisabled }}
             bodyStyles={{ overflow: 'hidden' }}
         >
-            <ModalContent>
-                <ModuleDetailsForm form={form} formValues={{ name: currentName }} />
-                <AssetsSection selectedAssetUrns={selectedAssetUrns} setSelectedAssetUrns={setSelectedAssetUrns} />
-            </ModalContent>
+            <ModalContext.Provider value={{ isInsideModal: true }}>
```

Comment:
even better, let's just move this to our `Modal` component we have in our component library so then we know we'll have this context in every modal! what do you think?